src/MDC/ModelDataCollector.cpp

```diff
@@ -332,6 +348,11 @@ void ModelDataCollector::perform_population_statistic() {
               blood_slide_prevalence_by_location_[loc] += 1;
               blood_slide_number_by_location_age_group_[loc][ac] += 1;
               blood_slide_number_by_location_age_group_by_5_[loc][ac1] += 1;
+              if (p->age() < 79) {
```

Comment:
Why 79 here but 80 above? Maybe the numbers should be converted to constants?

Reply:
Because some data has age class range from 0 to 80 while people's age is more than 80 so those people will be counted as the last index of age class, that's what the if statement did.